src/vs/editor/contrib/unicodeHighlighter/browser/unicodeHighlighter.ts

```diff
@@ -496,7 +496,7 @@ export class UnicodeHighlighterHoverParticipant implements IEditorHoverParticipa
 
 			const adjustSettings = nls.localize('unicodeHighlight.adjustSettings', 'Adjust settings');
 			const contents: Array<IMarkdownString> = [{
-				value: `${reason} [${adjustSettings}](command:${ShowExcludeOptions.ID}?${encodeURIComponent(JSON.stringify(adjustSettingsArgs))})`,
+				value: `${reason} [${adjustSettings}](command:${ShowExcludeOptions.ID}?${fixedEncodeURIComponent(JSON.stringify(adjustSettingsArgs))})`,
```

Comment:
Hmm, the issue actually is that the escaping conflicts with markdown.
I think the correct fix would be to use this helper:

https://github.com/microsoft/vscode/blob/30e1c0cfdaa3d1b3e7d4a0141bb8b0d3bcb51887/src/vs/base/common/htmlContent.ts#L78

Sorry for only noticing now.

Reply:
Replaced with `MarkdownString`